### PR TITLE
Added fifth class

### PR DESCRIPTION
**Pull request by AdarshShrivastava**
Hi AdarshLead

committed fifth class.

Regards
Adarsh

### CODE REVIEW

File: src/test/FifthClass.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package test;
+
+public class FifthClass {
+
+	public static void main(String[] args) {
+
```
Comment (adarsh2nd) [inline src/test/FifthClass.java:6]: Add few more code here
Add atleast some functionality

File: src/test/FifthClass.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+System.out.println("This is another class");
+	}
+
```
Comment (adarsh2nd) [inline src/test/FifthClass.java:9]: codes missing here

**Review by adarsh2nd — COMMENTED** [review]
Hi Adarsh

Please make the changes suggested.

Regards
Adarsh

File: src/test/FifthClass.java (hunk 1 above)
Comment (AdarshShrivastava) [reply src/test/FifthClass.java:6]: changes made

File: src/test/FifthClass.java (hunk 2 above)
Comment (AdarshShrivastava) [reply src/test/FifthClass.java:9]: this change also made

**Review by AdarshShrivastava — COMMENTED** [review]
Hi AdarshLead

I made the changes suggested.

Regards
Adarsh

**Pull request merged**